Apps/RaptureIntTests/src/test/java/rapture/api/checkout/SearchApiIntegrationTest.java

```diff
@@ -0,0 +1,309 @@
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package rapture.api.checkout;
+
+import static org.testng.AssertJUnit.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import com.google.common.net.MediaType;
+
+import rapture.common.CallingContext;
+import rapture.common.RaptureURI;
+import rapture.common.Scheme;
+import rapture.common.SearchHit;
+import rapture.common.SearchResponse;
+import rapture.common.SeriesPoint;
+import rapture.common.client.HttpBlobApi;
+import rapture.common.client.HttpDocApi;
+import rapture.common.client.HttpLoginApi;
+import rapture.common.client.HttpSearchApi;
+import rapture.common.client.HttpSeriesApi;
+
+/**
+ * Tests to exercise the Mongo repo to check for breakages in migrating to Mongo 3.0
+ */
+
+public class SearchApiIntegrationTest {
+
+    private HttpLoginApi raptureLogin = null;
+    private HttpSeriesApi seriesApi = null;
+    private HttpSearchApi searchApi = null;
+    private HttpDocApi docApi = null;
+    private HttpBlobApi blobApi = null;
+    CallingContext callingContext = null;
+    boolean cleanUpPrevious = true;
+    IntegrationTestHelper helper;
+
+    /**
+     * Setup TestNG method to create Rapture login object and objects.
+     *
+     * @param RaptureURL
+     *            Passed in from <env>_testng.xml suite file
+     * @param RaptureUser
+     *            Passed in from <env>_testng.xml suite file
+     * @param RapturePassword
+     *            Passed in from <env>_testng.xml suite file
+     * @return none
+     */
+    @BeforeClass(groups = { "mongo" })
```

Comment:
the execution groups need to be co-ordinated with Jeremy.  Cant have different groups across the test classes.

What is the thought on running the tests against the various Rapture configurations e.g. FTS off or on